paper/paper.md

```diff
@@ -4,20 +4,28 @@ title: 'ROSS - Rotordynamic Open Source Software'
 tags:
   - rotordynamic
   - numerical calculus
+  - Python
 authors:
- - name: Raphael Timbo
+ - name: Raphael Timbó
+   orcid: 0000-0000-0000-0000
    affiliation: 1
- - name: Thiago Ritto
-   affiliation: 2
- - name: Juliana Vianna
+ - name: Rodrigo Martins
+   orcid: 0000-0000-0000-0000
    affiliation: 2
  - name: Júlia Araújo
+   orcid: 0000-0000-0000-0000
    affiliation: 2
- - name: Rodrigo Martins
+ - name: Gabriel Bachman
+   orcid: 0000-0000-0000-0000
    affiliation: 2
  - name: Flavio Rangel
+   orcid: 0000-0002-4852-8141
    affiliation: 2
- - name: Gabriel Bachman
+ - name: Juliana Vianna
+   orcid: 0000-0000-0000-0000
+   affiliation: 2
+ - name: Thiago Ritto
+   orcid: 0000-0000-0000-0000
```

Comment:
It's not exactly a change request. It's only to remember the members to send their orcid numbers to add in here.

Reply:
Here is my number: 0000-0001-7689-5486

Reply:
Here is mine: 0000-0002-9996-6600

Reply:
My number is 0000-0002-9547-9629

Reply:
My number is 0000-0003-4401-8408